lib/roo/excelx/relationships.rb

```diff
@@ -19,6 +19,12 @@ def include_type?(type)
         end
       end
 
+      def target(type)
```

Comment:
`target` is a confusing name for what this method is doing.

We also need relevant test cases for the newly added method.

Reply:
The method name should signify that we are returning **target(s)** while filtering relations on **Type** attribute. If it doesn't signify the above then it is confusing in nature.